courses/free-courses-hi.md

```diff
@@ -99,6 +100,12 @@
 * [Web Development Tutorials for Beginners in Hindi: HTML, CSS, JavaScript and more](https://www.youtube.com/playlist?list=PLu0W_9lII9agiCUZYRsvtGTXdxkzPyItg) - CodeWithHarry
 
 
+#### Bootstrap
+
+* [Bootstrap Tutorial In Hindi](https://youtu.be/vpAJ0s5S2t0) - CodeWithHarry
```

Comment:
We do not permit shortened links. (youtube.com, not youtu.be)
Please read CONTRIBUTING.md guidelines!

Reply:
that's my mistake

Reply:
@GH05T-HUNTER5 could you add it again in #7005? Therefore we can avoid merge conflicts in TOC and Boostrap section

Reply:
Add there the next line 106 too

Reply:
Resolved by commit 8b7ae419ae60a37a755c9a91ed406fc7c54b7030 part of PR #7006